### PR TITLE
Consistent dry-run flag

### PR DESCRIPTION
**Pull request by kargakis**
Signed-off-by: Michail Kargakis <mkargaki@redhat.com>

### CODE REVIEW

**Comment by krzyzacy** [comment]
also bump hook

**Comment by kargakis** [comment]
@krzyzacy done - do you want me to also bump the version in the deployment manifest?

File: prow/cmd/hook/main.go [hunk 1]
```diff
@@ -51,8 +51,8 @@ var (
 	configPath   = flag.String("config-path", "/etc/config/config", "Path to config.yaml.")
 	pluginConfig = flag.String("plugin-config", "/etc/plugins/plugins", "Path to plugin config file.")
 
-	local = flag.Bool("local", false, "Run locally for testing purposes only. Does not require secret files.")
-	dry   = flag.Bool("dry", false, "Dry run for testing. Uses API tokens but does not mutate.")
+	local  = flag.Bool("local", false, "Run locally for testing purposes only. Does not require secret files.")
+	dryRun = flag.Bool("dry-run", false, "Dry run for testing. Uses API tokens but does not mutate.")
```
Comment (spxtr) [inline prow/cmd/hook/main.go]: At least for plank, the default is true. If you set it to true, be sure to manually set it to false in the deployment.

**Comment by spxtr** [comment]
LGTM with a comment.

File: prow/cmd/hook/main.go (hunk 1 above)
Comment (kargakis) [inline prow/cmd/hook/main.go]: ok, synced both the default values and the manifests, ptal

**Pull request merged**